src/bin/build.rs

```diff
@@ -43,6 +44,7 @@ Options:
     -h, --help                   Print this message
     -p SPEC, --package SPEC ...  Package to build
     --all                        Build all packages in the workspace
+    --exclude SPEC ...           Exclude packages from the build
```

Comment:
It seems pretty verbose to manually handle conflicts / requirements of flags. Is there a way to let `docopt` do this? There is currently sort of a bug when executing `cargo build -p whatever --all`: The `-p` is silently ignored.